src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2237,6 +2237,49 @@
      * The {@code default} keyword.  This element has no
      * children.
      *
+     * <p>For example:</p>
+     * <pre>
+     * switch (type){
```

Comment:
you can use checkstyle tool to validate your example code for code conventions

you should fix minor syntax violations:
- add space after `switch (type)`
- remove space after `case 1`
- add spaces around `=` in `x=1` and `x=3`

Reply:
I am using checkstyle plugin but it excludes comments I have done related changes. @timurt